server/api/src/rsDataObjTrim.cpp

```diff
@@ -54,42 +56,76 @@ namespace
         return info;
     }
 
-    int get_time_of_expiration(const char* age_kw) {
+    auto get_time_of_expiration(const char* age_kw) -> std::uint64_t
```

Comment:
Let's rename `age_kw` so it doesn't confuse the reader into thinking it's expected to be a keyword.

Reply:
Done